### PR TITLE
Add NFRs to the DG

### PR DESCRIPTION
**Pull request by wrewsama**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -335,9 +335,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+1. Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
+2. Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
+3. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+4. Should be able to load pages within 2 seconds.
```
Comment (Tex-Tang) [inline docs/DeveloperGuide.md:341]: Would it be better if we use `load the contact list` since `page` might be confused to users?

**Review by Tex-Tang — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (wrewsama) [reply docs/DeveloperGuide.md]: Okay that's a good idea. Updated.

**Review by Tex-Tang — APPROVED** [review]
LGTM!

**Review by sh4nH — APPROVED** [review]
lgtm

**Pull request merged**